conf/logs.json

```diff
@@ -308,5 +308,524 @@
         }
       }
     }
-  }
-}
\ No newline at end of file
+  },
+  "bro:conn.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "proto": "string",
+      "service": "string",
+      "duration": "string",
+      "orig_bytes": "integer",
```

Comment:
orig_bytes is of type `count`: https://www.bro.org/sphinx/script-reference/types.html#type-count, which supports hex notation like `0xff` or `0xABC123`.

This means any field that is of type `count` must be labelled as a `string`. Make this change for any field of type `count`